experimental/packages/opentelemetry-instrumentation-grpc/src/grpc-js/index.ts

```diff
@@ -298,8 +306,15 @@ export class GrpcJsInstrumentation extends InstrumentationBase {
             [SemanticAttributes.RPC_METHOD]: method,
             [SemanticAttributes.RPC_SERVICE]: service,
           });
+
+        instrumentation._metadataCapture.client.captureRequestMetadata(span, (metadataKey: string) => {
+          const metadataMap = metadata.getMap();
```

Comment:
We should be able to retrieve the metadata value by `metadata.get(key)`.

```suggestion
          return metadata.get(metadataKey);
```

Reply:
No need for this map after getting rid of the metadata getter